File: 75-sort-colors/sort-colors.cs
public class Solution {
    public void SortColors(int[] nums) {
        // Brute Force
        int idx = 0;
        for(int i=0; i<nums.Length; i++) {
            for(int j=i+1; j<nums.Length; j++) {
                if(nums[i] > nums[j]) {
                    int temp = nums[i];
                    nums[i] = nums[j];
                    nums[j] = temp;
                }
            }
        }
    }
}

File: 75-sort-colors/sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int i=nums.length - 1;
        int c = 2;
        while(i>=0 && c>=0) {
            for(int j=i; j>=0; j--) {
                if(nums[j]==c) {
                    int temp = nums[i];
                    nums[i] = nums[j];
                    nums[j] = temp;
                    i--;
                }
            }
            c--;
        }
    }
}